test/cmdlineTests/viair_subobjects/args

```diff
@@ -0,0 +1 @@
+--ir-optimized --experimental-via-ir --optimize --bin --bin-runtime
```

Comment:
If you add `--metadata` here, we can test the metadata.

Reply:
But is that properly trimmed (the version for example) in the commandline tests?

Reply:
Apparently not.